src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Tab } from '@mui/material';
+import Tabs from '@mui/material/Tabs';
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+
+export const AuthHeader = () => {
+  const location = useLocation();
+
+  return (
+    <Box sx={{ mt: 2, width: '100%' }}>
```

Comment:
My mentor told me that its better to use some css instead of sx prop. Personally I use css modules